### PR TITLE
Adding proposed volumesnapshot charts

### PR DESCRIPTION
**Pull request by drekle**
Just need to regenerate the index and package up charts after some other PR's merge.  

If at all possible I would like to keep my chart for snapshotPolicy with the source.

### CODE REVIEW

**Comment by drekle** [comment]
We should upstream the external storage pieces into incubator as well.

File: src/volumesnapshot/templates/clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - customresourcedefinitions
+    verbs:
+      - "create"
+  # Provisioner only
+  - apiGroups:
+      - "storage.k8s.io"
+    resources:
+      - storageclasses
+    verbs:
+      - list
```
Comment (wozniakjan) [inline src/volumesnapshot/templates/clusterrole.yaml:30]: I think there are couple of rules missing, see:
https://github.com/drekle/charts-1/pull/1

Comment (drekle) [reply src/volumesnapshot/templates/clusterrole.yaml:30]: merged, ty

**Review by wozniakjan — APPROVED** [review]

**Comment by wozniakjan** [comment]
@drekle I think you will need to rebase on master and re-run awake

**Review by ghost — APPROVED** [review]

**Pull request merged**